include/flecs/addons/http.h

```diff
@@ -126,6 +126,8 @@ typedef struct {
     uint16_t port;                    /**< HTTP port */
     const char *ipaddr;               /**< Interface to listen on (optional) */
     int32_t send_queue_wait_ms;       /**< Send queue wait time when empty */
+    ecs_ftime_t cache_timeout;             /**< Cache invalidation timeout (0 disables caching) */
+    ecs_ftime_t cache_purge_timeout;       /**< Cache purge timeout (for purging cache entries) */
```

Comment:
Is it worth noting that these are measured in seconds, and that the default for the `cache_purge_timeout` is 10x the `cache_timeout`?